app/models/form.rb

```diff
@@ -41,6 +41,11 @@ def questions_for_subsection(subsection)
     pages_for_subsection(subsection).map { |title, _value| questions_for_page(title) }.reduce(:merge)
   end
 
+  # Returns a hash with soft validation questions as keys
+  def soft_validations_for_page(page)
```

Comment:
Comment is really there for consistency with the methods above - it's about return type (hash) rather than what the thing is. Could add that to the method name I guess but possibly a bit clunky?

Reply:
In that case no need to change it 🙂